aws_lambda_powertools/utilities/parser/models/iot_registry_events.py

```diff
@@ -0,0 +1,110 @@
+import enum
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel
+
+
+class IOTCoreRegistryEventBase(BaseModel):
+    eventType: str
+    eventId: str
+    timestamp: int
+
+
+class IoTCRUDEventOperation(str, enum.Enum):
+    CREATED = "CREATED"
+    UPDATED = "UPDATED"
+    DELETED = "DELETED"
+
+
+class AddRemoveOperation(str, enum.Enum):
+    ADDED = "ADDED"
+    REMOVED = "REMOVED"
```

Comment:
```suggestion
EVENT_CRUD_OPERATION = Literal["CREATED", "UPDATED", "DELETED"]
EVENT_ADD_REMOVE_OPERATION = Literal["ADDED", "REMOVED"]
```

Please remove these `Enum` and transform them into a Literal type. It is easy to read and we don't need any type transformations to access the data. Also, move them to after the `import` section.